Doctors-MSSC/src/main/java/com/clinics/doctors/ui/controller/DoctorAppointmentController.java

```diff
@@ -0,0 +1,76 @@
+package com.clinics.doctors.ui.controller;
+
+import com.clinics.common.DTO.request.outer.doctor.AddEditAppointmentDTO;
+import com.clinics.common.DTO.response.outer.AppointmentResponseDTO;
+import com.clinics.doctors.ui.service.AppointmentService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+import java.util.UUID;
+
+@Controller
+@RequestMapping(value = "/doctors/{doctorUUID}/calendars/{calendarUUID}/appointments")
+public class DoctorAppointmentController {
+
+	final private AppointmentService appointmentService;
+
+	public DoctorAppointmentController(
```

Comment:
Nice injection :)

Reply:
rally or you laughing at me? :), we don't need to use @Autowired because spring managing this for as.  I even read about it :)